src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/TagList.netcore.cs

```diff
@@ -0,0 +1,340 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace System.Diagnostics
+{
+    /// <summary>
+    /// Represents a list of tags that can be accessed by index. Provides methods to search, sort, and manipulate lists.
+    /// </summary>
+    /// <remarks>
+    /// TagList can be used in the scenarios which need to optimize for memory allocations. TagList will avoid allocating any memory when using up to eight tags.
+    /// Using more than eight tags will cause allocating memory to store the tags.
+    /// Public static (Shared in Visual Basic) members of this type are thread safe. Any instance members are not guaranteed to be thread safe.
+    /// </remarks>
+    [StructLayout(LayoutKind.Sequential)]
```

Comment:
@stephentoub I'm missing the reason why this is necessary.

Reply:
It's not. It just came along as part of the copy from from the original code and I neglected to delete it.